presto-main/src/main/java/com/facebook/presto/memory/QueryContext.java

```diff
@@ -352,6 +354,7 @@ private String getAdditionalFailureInfo(long allocated, long delta)
         }
 
         String topConsumers = queryAllocations.entrySet().stream()
+                .filter(entry -> !entry.getKey().equals(FORCE_FREE_TAG))
```

Comment:
Another way to do this (if it's not proper to import `AbstractAggregatedMemoryContext.FORCE_FREE_TAG` directly, is to make sure `queryAllocations` returned by `memoryPool.getTaggedMemoryAllocations().get(queryId)` do not include this entry.

Reply:
cc @nezihyigitbasi

Reply:
Just chat with @nezihyigitbasi and we agreed that we should just filter this entry out in `MemoryPool.getTaggedMemoryAllocations`.